### PR TITLE
Four new pAI chassis

### PR DESCRIPTION
**Pull request by scarletflared**
Ah, there's the button for pull requests. If everything works like I think it does, this adds four (three and a half?) new pAI chassis for the five people that actually play pAIs.
paipreview

### CODE REVIEW

**Comment by Very-Soft** [comment]
As one of the 5 pAI players, HYPE HYPE HYPE

File: code/modules/mob/living/silicon/pai/pai.dm [hunk 1]
```diff
@@ -36,6 +36,10 @@
 		"Parrot" = "pai-parrot",
 		"Rabbit" = "pai-rabbit",
 		//VOREStation Addition Start
+		"Dire wolf" = "pai-diredog"
+		"Horse (Lune)" = "pai-horse_lune"
```
Comment (Very-Soft) [inline code/modules/mob/living/silicon/pai/pai.dm:40]: Ah, you need to ass commas at the end of list entries like the others

File: code/modules/mob/living/silicon/pai/pai_vr.dm [hunk 2]
```diff
@@ -48,6 +48,10 @@
 		"typeone",
 		"13",
 		"pai-raptor"
+		"pai-diredog"
```
Comment (Very-Soft) [inline code/modules/mob/living/silicon/pai/pai_vr.dm:51]: Here too

**Review by Very-Soft — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/silicon/pai/pai.dm (hunk 1 above)
Comment (scarletflared) [reply code/modules/mob/living/silicon/pai/pai.dm:40]: Ah good catch, I had the feeling I was missing something really obvious like that.

**Comment by SatinIsle** [comment]
These looks excellent

**Comment by scarletflared** [comment]
With any luck, the commas were the only thing I was missing.

**Pull request merged**